template/README.md

```diff
@@ -49,3 +49,28 @@ For `release` builds, we need to provide release keystore and signing properties
 
 - Put the `release.keystore` file at root `config` folder.
 - Put keystore signing properties in `signing.properties`
+
+## License
+
+This project is Copyright (c) 2014 and onwards Nimble. It is free software and may be redistributed under the terms specified in the [LICENSE] file.
+
+[LICENSE]: /LICENSE
+
+## About
+<a href="https://nimblehq.co/">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://assets.nimblehq.co/logo/dark/logo-dark-text-160.png">
+    <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-160.png">
+  </picture>
+</a>
+
+This project is maintained and funded by Nimble.
+
+We ❤️ open source and do our part in sharing our work with the community!
+See [our other projects][community] or [hire our team][hire] to help build your product.
+
+Want to join? [Check out our Jobs][jobs]!
+
+[community]: https://github.com/nimblehq
+[hire]: https://nimblehq.co/
+[jobs]: https://jobs.nimblehq.co/
```

Comment:
I personally think we should add `License` and `About` sections into the `root` template repository README file. W should NOT put them in `template` project. The reason is:
- The `template` project will be used to generate the project so keeping the `License` and `About` does not make sense. Nimble does not own the generated project for clients or other developers. If we agree to keep them, the developer will need to remove them later after generating. 🤔 
- However, we could keep them in `samples` because they're Nimble's own projects. 💭 

@nimblehq/android-chapter please let me know your ideas 🤓 

---

_FYI: please don't confuse with [my previous comment ](https://github.com/nimblehq/android-templates/pull/310#discussion_r972586972) to update the code block in all places. It's because I saw you put the dark logo in all places so we should make them all support dark/light mode, not to ask to put the License and About section in all places. 😛 _

> _@lydiasama Hmm, why do we remove this block for light mode? I think we should apply this for all places? 🤔_

Reply:
@luongvo That's a good point. Let me update.